jd-geek-source/src/app/components/test-diplay/test-diplay.component.html

```diff
@@ -0,0 +1,18 @@
+<div>
+  <h4>Testing Getting Data From The Best Buy API</h4>
+
+  <div *ngFor="let product of testResults">
+    <div>
+      <p>
+        <b>Department: </b>{{ product.department }}&nbsp;&nbsp; <b>SKU: </b
+        >{{ product.sku }}
+      </p>
+      <p><b>Name: </b>{{ product.name }}</p>
+      <p><b>Sale Price: </b>{{ product.salePrice }}</p>
+    </div>
+  </div>
+
+  <div>
+    <button [routerLink]="['/']">Home</button>
```

Comment:
I'd suggest using a custom button component instead of relying on the browser's native button. This will make maintenance much easier and provide a better means to ensure UI consistency throughout your app.